frontend/components/panels/VaultLiquidationLimitsCheckPanel.vue

```diff
@@ -180,7 +183,7 @@ export default Vue.extend({
     },
     methods: {
         format(value: BigNumber): string {
-            return formatToAutomaticDecimalPoints(value);
+            return formatWithThousandSeparators(formatToAutomaticDecimalPoints(value));
```

Comment:
I would use `formatWithThousandSeparators` inside `formatToAutomaticDecimalPoints` and make it accept second parameter with options `formatToAutomaticDecimalPoints(value, { disableThousandSeparators: true })`